evalml/model_understanding/graphs.py

```diff
@@ -653,6 +652,11 @@ def partial_dependence(
             is_datetime = [_is_feature_of_type(features, X, ww.logical_types.Datetime)]
 
         if isinstance(features, (list, tuple)):
+            if any(is_datetime) and len(features) > 1:
```

Comment:
There should to be two `if isinstance(features, (list, tuple))` checks. Consolidating into one now.